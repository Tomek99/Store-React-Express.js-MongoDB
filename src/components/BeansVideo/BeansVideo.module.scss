.VideoDetails {
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;

    .test {
        width: 750px;
        height: 500px;
    }
}
