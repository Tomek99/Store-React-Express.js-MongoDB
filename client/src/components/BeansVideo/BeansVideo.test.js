import { render, screen } from "@testing-library/react";
import BeansVideo from "./BeansVideo";

test("Should render BeansVideo component", () => {
  const { container } = render(<BeansVideo />);

  expect(container).toMatchSnapshot();
});
