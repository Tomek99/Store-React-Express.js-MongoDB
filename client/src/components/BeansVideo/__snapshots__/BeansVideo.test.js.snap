// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render BeansVideo component 1`] = `
<div>
  <div
    class="VideoDetails"
  >
    <video
      autoplay=""
      class="test"
      controls=""
      controlslist="nodownload"
      loop=""
    >
      <source
        src="./videos/beans.mp4"
        type="video/mp4"
      />
    </video>
  </div>
</div>
`;
